### ISSUE TITLE
<img onError={} /> should not trigger no-noninteractive-element-interactions rule

### ISSUE THREAD

**Issue by echenley** [evcohen/eslint-plugin-jsx-a11y#309]
The following causes eslint to error on the jsx-a11y/no-noninteractive-element-interactions rule.

```js
<img onError={() => {}} />
```
